test.diff

```diff
@@ -0,0 +1,20 @@
+diff --git a/app/src/main/java/org/schabi/newpipe/player/VideoPlayer.java b/app/src/main/java/org/schabi/newpipe/player/VideoPlayer.java
```

Comment:
Huh :smile: 
The diff looks a little lost here :D

Reply:
ssht. Nothing to see here :D but you might want to check #2569 as another nice little addition for Android9+ devices

Reply:
I didn't see anything :D
I'll take a look at #2569 soon. There are so many open PRs at the moment... I need more time and help to review them all

Reply:
PS: you can squash the commits and nobody will ever know about that file :)